test/e2e/tests/request-queuing/ui.spec.js

```diff
@@ -116,17 +115,10 @@ async function switchToDialogPopoverValidateDetailsRedesign(
 }
 
 async function rejectTransactionRedesign(driver) {
-  if (isManifestV3) {
```

Comment:
I'm unsure why this was added, but this cause MV3 runs to flaky. Here is what happens:
1. We have a dialog A open
2. We click Cancel
3. We don't wait until the dialog A is closed
4. We trigger a new tx, which will open a new dialog, but:
5. We switch to the dialog window -- it's dialog A, instead of B
6. Dialog A is now closed due to the step 2 action
7. We get error, window already closed